Entity/PostManager.php

```diff
@@ -96,7 +96,10 @@ public function getPager(array $criteria, $page, $limit = 10, array $sort = arra
         $query = $this->getRepository()
             ->createQueryBuilder('p')
             ->select('p, t')
+            ->andWhere('p.publicationDateStart <= :now')
             ->orderBy('p.publicationDateStart', 'DESC');
+        
+        $parameters['now'] = new \DateTime();
```

Comment:
Isn't doctrine able to map to a different method? See https://github.com/doctrine/dbal/search?utf8=%E2%9C%93&q=getNowExpression&type=